client/src/components/Upload/UploadModalContent.vue

```diff
@@ -238,6 +271,38 @@ export default {
             }
             return data;
         },
+        toFetchData: function (items, history_id) {
+            var data = {
+                history_id: history_id,
+                targets: [
+                    {
+                        destination: { type: "hdas" },
+                        elements: [],
+                        name: "",
+                    },
+                ],
+                auto_decompress: true,
+            };
+
+            // Composite does not use the fetch API, so we can just
+            // index into the first element of items
+            const urls = items[0].get("url_paste").split("\n");
+            const dbkey = items[0].get("genome", "?");
+            const ext = items[0].get("extension", "auto");
```

Comment:
I assume these need to get added also:

```
                        inputs[`${prefix}space_to_tab`] = (it.get("space_to_tab") && "Yes") || null;
                        inputs[`${prefix}to_posix_lines`] = (it.get("to_posix_lines") && "Yes") || null;
```

They are used in data_fetch.py and they are probably just treated as booleans (instead of "Yes" or "" strings).

```
            to_posix_lines = upload_config.get_option(item, "to_posix_lines")
            space_to_tab = upload_config.get_option(item, "space_to_tab")
```

The defaults might be different also:

```
        self.to_posix_lines = request.get("to_posix_lines", False)
        self.space_to_tab = request.get("space_to_tab", False)
```